tests/mocks/hashes.txt

```diff
@@ -1,4 +1,4 @@
 a981b9a2fff3d75a6b3b8683c263f993
 5f4dcc3b5aa765d61d8327deb882cf99
 NWY0ZGNjM2I1YWE3NjVkNjFkODMyN2RlYjg4MmNmOTk=
-password
\ No newline at end of file
+password

```

Comment:
I don't see any new, blank lines here testing for Hydra's bug.

Reply:
Blank lines should be alredy in there, but Ill add one to the top.